usr/src/uts/i86pc/io/vmm/vmm.c

```diff
@@ -1193,9 +1193,13 @@ vm_gpa_release(void *cookie)
 {
 	vm_page_t m = cookie;
 
+#ifdef __FreeBSD__
```

Comment:
It doesn't really make sense to do this - now we have no other users of the vm_page_lock/unlock/hold(), we can simply remove our implementation in favour of a vm_page_unwire() which does the same thing but merged into one function.

Reply:
done